.github/workflows/build.yml

```
@@ -65,7 +65,7 @@ jobs:
         with:
           tool: cross
       - run: mkdir dist
-      - run: ${{ (!matrix.os && matrix.use-cross) && 'cross' || 'cargo' }} rustc --locked --release --no-default-features --target=${{ matrix.target }} -- --emit=link=dist/binary
+      - run: ${{ (!matrix.os && matrix.use-cross) && 'cross' || 'cargo' }} rustc --locked --release --features webpki --no-default-features --target=${{ matrix.target }} -- --emit=link=dist/binary
```

```suggestion
      - run: ${{ (!matrix.os && matrix.use-cross) && 'cross' || 'cargo' }} rustc --locked --release --features=webpki --no-default-features --target=${{ matrix.target }} -- --emit=link=dist/binary
```
等号を使うとオプションに渡している値であることが明確になる